core/templates/pages/exploration-editor-page/statistics-tab/issues/playthrough-issues.component.html

```diff
@@ -42,3 +42,11 @@
     </div>
   </div>
 </div>
+<style>
+  playthrough-issues .oppia-display-block {
+    display: block;
+  }
+  playthrough-issues .oppia-display-none {
+    display: none;
+  }
```

Comment:
Use the bootstrap classes

Reply:
I've removed the classes as per Brian's comment.